Notebooks/Bruxism_detection.ipynb

```diff
@@ -2,24 +2,23 @@
  "cells": [
```

Comment:
<p>hardcoded parameters for interval the size: 50, 50 which is problematic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>